stdlib/unittest/case.pyi

```diff
@@ -4,12 +4,12 @@ import sys
 import unittest.result
 from types import TracebackType
 from typing import (
+    AbstractSet,
```

Comment:
Instead of using `AbstractSet`, we should use `collections.abc.Set`. Using `collctions.abc` is now possible in typeshed and the preferred way.